src/main/java/com/redhat/devtools/lsp4ij/LSPIJEditorUtils.java

```diff
@@ -147,27 +146,27 @@ private static Set<Character> getAbstractFileTypeQuoteCharacters(@NotNull PsiFil
 
     @Nullable
     private static Set<Character> getTextMateQuoteCharacters(@NotNull PsiFile file) {
-        /* TODO: See TextMate versioning comment above
-        Editor editor = isTextMateFile(file) ? LSPIJUtils.editorForElement(file) : null;
-        TextMateScope scope = editor instanceof EditorEx editorEx ? TextMateEditorUtils.getCurrentScopeSelector(editorEx) : null;
-        List<Preferences> allPreferences = scope != null ? TextMateService.getInstance().getPreferenceRegistry().getPreferences(scope) : Collections.emptyList();
-        Preferences preferences = ContainerUtil.getFirstItem(allPreferences);
-        Set<TextMateAutoClosingPair> smartTypingPairs = preferences != null ? preferences.getSmartTypingPairs() : Collections.emptySet();
-        if (!ContainerUtil.isEmpty(smartTypingPairs)) {
-            Set<Character> quoteCharacters = new LinkedHashSet<>();
-            for (TextMateAutoClosingPair smartTypingPair : smartTypingPairs) {
-                CharSequence left = smartTypingPair.getLeft();
-                CharSequence right = smartTypingPair.getRight();
-                if ((left.length() == 1) && left.equals(right)) {
-                    char quoteCharacterCandidate = left.charAt(0);
-                    if (DEFAULT_QUOTE_CHARACTERS.contains(quoteCharacterCandidate)) {
-                        quoteCharacters.add(quoteCharacterCandidate);
+        if (isTextMateFile(file)) {
```

Comment:
This check ensures that classes from the **TextMate Bundles** plugin will _not_ be evaluated at runtime if that plugin is disabled.